prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,52 @@
+"""
+CP1404 Week 8 Practical
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+CONVERSION_VALUE = 1.609344
+
+
+class ConvertMilesKm(App):
+    """ ConvertMilesKm is a Kivy App for converting miles to km """
+    message = StringProperty()
+
+    def build(self):
+        """ build the Kivy app from the kv file """
```

Comment:
I think doc strings should start with a capital letter and end with a full stop.